packages/salesforcedx-slds-linter/src/messages/i18n.ts

```diff
@@ -17,15 +17,15 @@
  */
 export const messages = {
   fix_problem:
-  `Fix: %s`,
+  `Fix this problem: %s`,
   fix_same:
-  `Fix all: %s`,
+  `Fix all instances of: %s`,
   fix_all:
   'Fix all auto-fixable problems',
   fix_error:
-  'Failed to apply SLDS Validator fixes to the document',
+  'Failed to apply SLDS Validator fixes to the document.',
```

Comment:
Do we have any idea what would have caused this problem? Can we suggest a solution? Also, I added a period, but if it displays in the same format as the other messages in this section we can remove the period. VS Code seems to use sentence-case capitalization and no end punctuation for these contextual messages.

Reply:
It will only fail if the edit function doesn't successfully run. This wouldn't be a problem that the end user can fix. One thing we can suggest is filing an issue on Github with us.

Reply:
It doesn't display along with the other messages, so the period should be okay